### PR TITLE
Major improvements

### PR DESCRIPTION
**Pull request by kevva**
Need this to make optipng work correctly on Windows.
- Windows support
- Consistent code style
- Better documentation

cc @ameyp

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+'use strict';
+
+var concat = require('concat-stream');
+var fs = require('fs');
+var optipng = require('optipng-bin').path;
+var rm = require('rimraf');
+var spawn = process.platform === 'win32' ? require('win-spawn') : require('child_process').spawn;
```
Comment (sindresorhus) [inline cli.js:8]: no need for conditional, normal spawn is used on unix already

**Comment by sindresorhus** [comment]
:+1:

**Comment by ameyp** [comment]
Looks great, thank you very much. However, I prefer four-space-indentation instead of two-space-indentation, so I'll convert that myself and then merge it.

**Comment by ameyp** [comment]
Merged with commit https://github.com/ameyp/optipng-stream-bin/commit/85b736665b3cbd56840342de0d7afc4ea5bc9641, thanks again.

**Pull request closed without merging**